### ISSUE TITLE
Allow multiple product lists for shifts

### ISSUE THREAD

**Issue by JobDoesburg** [svthalia/concrexit#1786]
### Is your feature request related to a problem? Please describe.
Among others because of #1654, it would be nice to be able to sell both pizzas and regular borrel products in a single shift

### Describe the solution you'd like
Let product_list be a ManyToMany field, and let each shift construct its own effective product list consisting of the union of all product lists that are selected

### Motivation
So both pizzas and beer can be sold in the same shift

### Describe alternatives you've considered
Creating product lists like 'borrel with pizza' and 'borrel without pizza'. Which works absolutely fine! So priority is maybe_someday

**Comment by se-bastiaan** [comment]
Doesn't this depend on the solution we choose for the food events?

**Comment by JobDoesburg** [comment]
> Doesn't this depend on the solution we choose for the food events?

Yes, but even without food events at all it could be a useful feature
